README.md

```diff
@@ -114,4 +114,59 @@ To run `DefaultAzureCredentailAdapterTest`, ensure you have `.env` file created
 
 Once you have the `.env` file configured, run the test using JUnit 5 runner.
 
+## Python
+
+### AzureIdentityCredentialWrapper
```

Comment:
Let's name it AzureIdentityCredentialAdapter instead of wrapper.  That aligns with what we are doing for Java as well.